src/backends/custom/loader.cc

```diff
@@ -72,17 +74,28 @@ LoadCustom(
   *ExecuteFn = nullptr;
   *ExecuteV2Fn = nullptr;
   *custom_version = 0;
+  Status status;
+
+  // Add handling for S3 path
+  // FIXME Add GCS Support
+  std::string local_path = "";
+  if (path.rfind("s3://", 0) == 0) {
+    status = DownloadFileFolder(path, &local_path);
```

Comment:
Why are you downloading files to local here instead of in the backend factory like for TF? Does the file need to be presented even after dlopen()? If so, remember to destroy the local folder in unload custom. If not, perhaps just follow the pattern for TF

Reply:
Fixed. Yes the file can be deleted after dlopen is called.